module.exports.constant = {
  /*---------------------------------------------[ error fillter handle ]----------------------------------------------*/
  HANDLING_ERROR_EXCEPTION: "An unexpected error occurred.",
  INTERNAL_SERVER_ERROR: "Internal Server Error",

  /*---------------------------------------------[ logger ]----------------------------------------------*/
  USER_TABLE_CREATE: "User Table Create in Database",

  /*---------------------------------------------[ Auth ]----------------------------------------------*/
  TOKEN_EMPTY: "No token or no Api Key,authorization denied",

  /*---------------------------------------------[ User ]----------------------------------------------*/
  USER_ALREADY_EXIST: `User already exists`,
  NO_SUCH_USER: `No such user found.`,
  WRONG_PASSWORD: "Wrong password",
  VERIFY_MAIL: "Verify Mail",
  CONFIRM_REGISTRATION: `Confirm your registration`,
  USER_REGISTER_SUCCESSFULLY: `User Register Successfully`,
  USER_ALREADY_VERIFY: "User already verified..",
  OTP_WRONG: "OTP Wrong",
  EMAIL_VERIFIED_SUCESSFULLY: "Email verified successfully",
  THIS_ACCOUNT_NOT_FOUND: "This account is not found",
  PLEASE_REGISTER: "Please Register First!",
  CREDENTIAL_WRONG: "Your Credential Wrong!",
  PLEASE_CONFIRM_REGISTRATION: "Please confirm your registration first!",
  USER_FOUND_SUCCESFULLY: "User found succesfully",
};
